releasenotes/notes/cleanup-curve-analysis-96d7ff706cae5b4e.yaml

```diff
@@ -0,0 +1,31 @@
+---
+developer:
```

Comment:
Some of this should go into `upgrade` notes if the API for how you subclass this has changed in a possibly breaking way.